packages/vuelidate/package.json

```diff
@@ -3,7 +3,7 @@
   "description": "Simple, lightweight model-based validation for Vue.js",
   "version": "2.0.2",
   "main": "dist/index.cjs",
-  "types": "index.d.ts",
+  "types": "dist/index.d.cts",
```

Comment:
Could changing the extension break older versions of typescript?

Reply:
Yeah, TS versions/configurations that understand `exports` (and the newer file extensions) won’t look at this top-level `types` field since they’ll look in `exports` instead, so it’s best practice to leave the top-level `types` as a `.d.ts` (as long as you have `exports`) for the benefit of old TS versions.

Reply:
Great, thanks for the clarification, that's really helpful to know.